### PR TITLE
Add intro screen

### PR DESCRIPTION
**Pull request by aks-**

### CODE REVIEW

**Comment by aks-** [comment]
closes https://github.com/dat-land/dat-desktop/issues/468

**Comment by martinheidegger** [comment]
The intro-screen PR seems to work but in the current version of Dat it is only shown when no dat has been added yet. In my test I wasn't able to determine if it works as every time I refreshed the app it would again start in a empty-state. Should we wait with this PR until it can be verified that it worked before, or is it okay to merge it in anticipation that it will work better after some other work is finished as well?

**Comment by juliangruber** [comment]
> In my test I wasn't able to determine if it works as every time I refreshed the app it would again start in a empty-state.

I was thinking, we can finish this PR but merge it into `react` until we solved persistence, otherwise it would be to annoying to work on it.

**Comment by juliangruber** [comment]
@aks- please run `npm test`, this will lint and format your code

**Comment by juliangruber** [comment]
@aks- `npm test` reported an error about a double import of `electron`, can you fix that too?

**Comment by aks-** [comment]
@juliangruber updated PR. Thanks for catching those :)

File: app/containers/table.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ import { connect } from 'react-redux'
 
 const mapStateToProps = state => ({
   dats: state.dats,
-  show: !state.inspect.key
+  show:
+    !state.inspect.key && (Object.keys(state.dats).length || !state.intro.show)
```
Comment (juliangruber) [inline app/containers/table.js]: this should be moved into the reducers

**Comment by juliangruber** [comment]
@aks- can you push your branch onto this repo as well? i can take a look at refactoring the reducer then if you're ok with it.

**Comment by aks-** [comment]
@juliangruber done, would you want me to do same with other new branches I push now on?

**Comment by juliangruber** [comment]
> @juliangruber done, would you want me to do same with other new branches I push now on?

Yes please, push them all to here :)

**Comment by juliangruber** [comment]
Closing in favor of https://github.com/dat-land/dat-desktop/pull/502

**Pull request closed without merging**